api/users/models.py

```diff
@@ -121,6 +122,28 @@ def __str__(self):
     def subscribe_to_mailing_list(self):
         mailer_lite.subscribe(self)
 
+    @hook(AFTER_UPDATE, when="email", has_changed=True)
+    def send_warning_email(self):
+        message = inspect.cleandoc(
+            f"""
```

Comment:
Can we move this to use a template instead of the string here? Also, let's use the user's `first_name` attribute instead of 'there' (since we should have it here).

And let's update the content slightly. Instead of saying 'Your new email address is {self}.' let's say: 

```
Hi {first_name},

A change was just made to update the email associated with your Flagsmith account. If you did not request this change, please contact support@flagsmith.com as soon as possible. 

The Flagsmith Team
```